src/shipping/components/ShippingZoneDetailsPage/ShippingZoneDetailsPage.tsx

```diff
@@ -202,6 +225,41 @@ const ShippingZoneDetailsPage: React.FC<ShippingZoneDetailsPageProps> = ({
                   warehouses={warehouseChoices}
                 />
               </div>
+              <TextField
+                error={description.length > maxDescriptionLength}
+                name={"description"}
+                label={
+                  <div className={classes.labelContainer}>
+                    <div className={classes.label}>
+                      <FormattedMessage defaultMessage="Description" />
+                    </div>
+                    {description?.length > 0 && (
+                      <span>
+                        <FormattedMessage
```

Comment:
Nothing big, but in this file there are a lot of `defaultMessage`. We can move all messages into `const messages = defineMessages({...})`